lib/shared/routers/public.js

```diff
@@ -3,22 +3,46 @@ import Public from 'screens/public';
 import React from 'react';
 import SchemaEntry from 'screens/public/screens/schema-entry';
 import {Route, IndexRoute} from 'react-router';
+import request from 'helpers/request';
+
+function initialized (nextState, replaceState, callback) {
```

Comment:
Also no need to use this for the public routes. The change should only be for the /admin/login route